src/components/dialogs/parameters/widget/parameter-line-slider.tsx

```diff
@@ -32,6 +32,12 @@ const ParameterLineSlider = ({
     const [parameterValue, handleChangeParameterValue] = useParameterState(paramNameId);
     const [sliderValue, setSliderValue] = useState(Number(parameterValue));
 
+    useEffect(() => {
+        if (parameterValue) {
+            setSliderValue(Number(parameterValue));
+        }
+    }, [parameterValue]);
```

Comment:
This is a fix for another scope ? I mean it's wasn't working before your development...
Maybe do it in another PR ?